### DISCUSSION TITLE
iLogtail开源社区报（第2期），内有惊喜，欢迎点击！

### DISCUSSION THREAD

**Discussion by messixukejia** [alibaba/ilogtail#379] [Announcements]
## iLogtail开源社区报（第2期）已发布

推荐阅读：
* [零信任策略下K8s安全监控最佳实践（K+）](https://link.zhihu.com/?target=https%3A//mp.weixin.qq.com/s/wYUNsGaWEnQZ0BVxsQORbA)
* [iLogtail开源之路](https://link.zhihu.com/?target=https%3A//mp.weixin.qq.com/s/vGGnHBV7l3ePwJgK4GxW6w)
* [您有一份来自iLogtail社区的礼物待查收](https://link.zhihu.com/?target=https%3A//mp.weixin.qq.com/s/fyWwnKR1I4jgNiX30Wu-Vg)

更多欢迎[查看全文](https://zhuanlan.zhihu.com/p/563246436)

## 社区回馈
第六届QECon全球软件质量&效能大会将于9月16-17日上海召开。阿里云SLS团队也将带来《测试数据模拟方案实践与探索》的主题分享。同时为了回馈iLogtail社区贡献者，我们免费送出一张QECon大会门票，欢迎下方留言“为iLogtail打Call”，我们将抽取一位朋友免费赠与门票。

![image](https://user-images.githubusercontent.com/13250519/189485715-43739906-ed74-4a7e-acde-6ed334e724bb.png)
大会链接：[QECon全球软件质量&效能大会上海站第六届](http://www.qecon.net/)

大会支持线上、线下两种模式，即使非上海的也可观看。
<img src="https://user-images.githubusercontent.com/13250519/189518216-e2195a49-49ac-40ff-aeb5-1d8666802499.png" height="50%" width="50%" >

**Comment by jiayounannan** [comment]
Want it!

**Comment by jiayounannan** [comment]
为iLogtail打Call

**Comment by 0xClownfish** [comment]
为iLogtail打Call！
上次meetup时讲到无侵入数据采集的部分老师能分享一下材料吗？

**Reply by messixukejia** [reply]
https://mp.weixin.qq.com/s/iEk1cQXWcJ1Qxr7Ab8bp5g

**Comment by sjtuzbk** [comment]
期待ilogtail不断发展
